recipes/slurmgen/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "slurmgen" %}
+{% set version = "3.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/slurmgen-{{ version }}.tar.gz
+  sha256: c1704dc4eda6f34ec11944165be05f7a70dfcd05c626e86730aabb3289e558f7
+
+build:
+  entry_points:
+    - sgen = slurmgen:run_script
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >68.2
+    - wheel >0.41
+    - setuptools-scm >7.1
+    - pip
+  run:
+    - python >=3.6
```

Should be the same as host.
```suggestion
    - python >=3.9
```

---

Thanks for pointing that. I've downgraded the host version to 3.6.